### PR TITLE
propose a couple of spelling corrections

### PR DESCRIPTION
**Pull request by jeis2497052**
##### ISSUE TYPE

 - Feature Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### COMPONENT NAME

 - API
 - UI
 - Installer

##### AWX VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: awx/locale/django.pot [hunk 1]
```diff
@@ -791,7 +791,7 @@ msgid "Request to {} timed out."
 msgstr ""
 
 #: awx/api/views.py:2129
```
Comment (ryanpetrello) [inline awx/locale/django.pot:793]: @shanemcd correct me if I'm wrong - changes to this `.pot` file should have corresponding changes in the `.py` source code files they reference.

Comment (jeis2497052) [reply awx/locale/django.pot:793]: sorry, let me add one more fix if that is OK ?

Comment (shanemcd) [reply awx/locale/django.pot:793]: This file should not be updated manually. It is generated from the source code. There are arguments for this file not being in source control at all, but that is a separate discussion.

Comment (jeis2497052) [reply awx/locale/django.pot:793]: my mistake.  Would this string change make sense then ?
```
diff --git a/awx/ui/conf.py b/awx/ui/conf.py
index 0d626c28f..577b9898f 100644
--- a/awx/ui/conf.py
+++ b/awx/ui/conf.py
@@ -58,7 +58,7 @@ register(
     field_class=fields.IntegerField,
     min_value=100,
     label=_('Max Job Events Retreived by UI'),
-    help_text=_('Maximum number of job events for the UI to retreive within a '+    help_text=_('Maximum number of job events for the UI to retrieve within a '                 'single request.'),
     category=_('UI'),
```

Comment (ryanpetrello) [reply awx/locale/django.pot:793]: @jeis2497052 yep, if you don't mind, feel free to update the corresponding `.py` file(s) like this.  But let's leave the `.pot` file unchanged - it's modified via external tooling.

Thank you 😄 !

**Comment by matburt** [comment]
re-submit PR without pot file changes

**Pull request closed without merging**